### ISSUE TITLE
[a11y]: Toggle has 2 violations, arial-abel and landmark role

### ISSUE THREAD

**Issue by hyungskim** [carbon-design-system/carbon#14920]
### Package

@carbon/react

### Package version

v10.58.10 (carbon-component) 7.59.14 (carbon-component-react)

### React version

16.14

### Automated testing tool and ruleset

IBM Equal Accessibility Checker (latest one)

### Description

There are 2 violations from the carbon Toggle component:

1 issue from the Toggle switch itself and the other was from the checker component that the Toggle switch uses internally.

**Violation 1:**

<img width="1856" alt="Screen Shot 2023-10-16 at 11 10 54 AM" src="https://github.com/carbon-design-system/carbon/assets/50377275/db93855e-466a-4361-96ba-dd72703bc745">

<img width="1068" alt="Screen Shot 2023-10-16 at 11 12 46 AM" src="https://github.com/carbon-design-system/carbon/assets/50377275/10d680ee-3d78-478b-989d-cc19de3cb926">

**Violation 2:**

<img width="1861" alt="Screen Shot 2023-10-16 at 11 13 31 AM" src="https://github.com/carbon-design-system/carbon/assets/50377275/789e267f-b81f-4858-a72a-1e0ab2a622c3">

<img width="1069" alt="Screen Shot 2023-10-16 at 11 15 37 AM" src="https://github.com/carbon-design-system/carbon/assets/50377275/d7310413-6834-45cd-b355-7ae537e47007">

### Reproduction/example

Reproducible using the <Toggle/>

### Steps to reproduce

1. Use <Toggle/> in the UI page.
2. Run Accessibility checker against the page.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by hyungskim** [comment]
Hi, @guidari  
I'm from Catalog UI IBM Knowledge Catalog. We've noticed 2 violations in the Toggle switch component.

**Comment by tw15egan** [comment]
@hyungskim for the first one, simply remove the `aria-label` prop from the component. If there is none and it is being auto-generated, then try updating to the latest version of V10. I am unable to reproduce this violation in the [storybook](https://v7-react.carbondesignsystem.com/iframe.html?id=components-toggle--default&args=&viewMode=story).

As far as the landmark violation, this is due to your app not having a valid `landmark` element. Usually, you can set the main wrapper of your app from a `div` to a `main` element. More information can be found here: https://developer.mozilla.org/en-US/blog/aria-accessibility-html-landmark-roles/

Hope that helps!
